### PR TITLE
Update/0.12

### PR DESCRIPTION
**Pull request by LiamGoodacre**
Does this business with `runSTRecord` and `pureSTRecord` look right?

### CODE REVIEW

File: src/Data/Record/ST.purs [hunk 1]
```diff
@@ -5,67 +5,62 @@ module Data.Record.ST
...
 
 -- | Run an ST computation safely, constructing a record.
-foreign import runSTRecord :: forall r eff. (forall h. Eff (st :: ST h | eff) (STRecord h r)) -> Eff eff (Record r)
-
--- | Run an ST computation safely, constructing a record, assuming no other
--- | types of effects.
-pureSTRecord :: forall r. (forall h eff. Eff (st :: ST h | eff) (STRecord h r)) -> Record r
-pureSTRecord st = runPure (runSTRecord st)
+runSTRecord :: forall r. (forall h. ST h (STRecord h r)) -> Record r
+runSTRecord st = run (st >>= freezeSTRecord)
```
Comment (garyb) [inline src/Data/Record/ST.purs]: Maybe this doesn't even need to exist anymore?

**Review by garyb — COMMENTED** [review]
We could update everything in the `ST` module to drop the `STRecord` prefix too, like we've done in a few other places, since it can be handled nicely with qualification anyway.

**Comment by LiamGoodacre** [comment]
@garyb I agreed with and have made the changes you mentioned 😺

**Review by garyb — APPROVED** [review]
Sorry, forgot this one was pending :)

**Pull request merged**